TASK/TaskGoToCoordWhileAimingAtEntity.md

```diff
@@ -5,23 +5,24 @@ ns: TASK
 
 ```c
 // 0xB2A16444EAD9AE47 0x9BD52ABD
-void TASK_GO_TO_COORD_WHILE_AIMING_AT_ENTITY(Any p0, float p1, float p2, float p3, Any p4, float p5, BOOL p6, float p7, float p8, BOOL p9, Any p10, BOOL p11, Any p12, Any p13);
+void TASK_GO_TO_COORD_WHILE_AIMING_AT_ENTITY(Ped ped, float x, float y, float z, Entity entityToAimAt, float moveSpeed, BOOL shoot, float targetRadius, float slowDistance, BOOL useNavMesh, int navFlags, BOOL instantBlendAtAim, Hash firingPattern, int time);
```

Comment:
Hey there, when changing types you're supposed to use `cs_type` so it doesn't cause ABI breakage. You can use [this](https://github.com/citizenfx/natives/blob/5d614f1dcc81e3faa744806535f9ee6a25dcbc50/PED/ClonePedEx.md?plain=1#L9) as an example.

And here's the [contribution guide](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md#changing-parameter-or-return-types), in case you missed it (specifically the section that mentions type changes) 😊

Thank you!

Reply:
Fixed, thanks for the help!